website/content/docs/commands/pki/health-check.mdx

```diff
@@ -325,6 +449,16 @@ This health check verifies that tidy has run within the last run window. This ca
 
 This health check verifies that this cluster has a reasonable number of certificates. Ideally this would be fetched from tidy's status or a new metric reporting format, but as a fallback when tidy hasn't run, a list operation will be performed instead.
 
+**Remediation Steps**:
+
+If a large amount of certificates have accumulated, verify that tidy has recently run using `vault read <mount>/tidy-status`. Consider
+enabling `auto-tidy` based on instructions above and running a manual tidy.
+
+Also ensure that certificates are not being renewed too soon based on lifetimes, roughly 2/3 of its lifetime, and
+certificates are being issued with appropriate lifetimes based on their usage.
+
+To help going forward, consider setting all roles' `no_store` fields to `true` and leverage [BYOC revocations](/vault/api-docs/secret/pki#revoke-certificate) to avoid storage.
```

Comment:
```suggestion
1. Verify that tidy ran recently with `vault read`:
    ```shell-session
    vault read <mount>/tidy-status
    ````
1. Schedule a manual run of tidy with `vault write`:
	    ```shell-session
	    vault write <mount>/tidy \
	      tidy_cert_store=true \
	      tidy_revoked_certs=true \
	      tidy_acme=true \
	      tidy_revocation_queue=true \
	      tidy_cross_cluster_revoked_certs=true \
	      tidy_revoked_cert_issuer_associations=true
	    ```
1. Enable `auto-tidy`.
1. Make sure that you are not renewing certificates too soon. Certificate
   lifetimes should reflect the expected usage of the certificate. If the TTL is
   set appropriately, most certificates renew at approximately 2/3 of their
   lifespan.
1. Consider setting the `no_store` field for all roles to `true` and use [BYOC revocations](/vault/api-docs/secret/pki#revoke-certificate) to avoid storage.
```